save-agent/src/commonMain/kotlin/com/saveourtool/save/agent/utils/Utils.kt

```diff
@@ -7,9 +7,8 @@ package com.saveourtool.save.agent.utils
 import com.saveourtool.save.agent.AgentConfiguration
 import com.saveourtool.save.agent.AgentEnvName
 import com.saveourtool.save.core.logging.logTrace
-import generated.SAVE_CORE_VERSION
 
-internal const val SAVE_CLI_EXECUTABLE_NAME = "save-$SAVE_CORE_VERSION-linuxX64.kexe"
+internal const val SAVE_CLI_EXECUTABLE_NAME = "save-linuxX64.kexe"
```

Comment:
Same question here. For me it seems useful to know which versions of all components have been used for a particular execution to make debugging easier.

Reply:
here (in file name) version is not required